app/routes/groups/add_user.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import RestUtils from 'moviematcher/utils/rest';
+
+export default Ember.Route.extend({
+  model(params) {
+    const group_name = params.group_name;
+    params['group_name'] = group_name;
+
+    console.log(params.data);
+
+    return RestUtils.get(this, `/v1/groups/${group_name}/user`)
```

Comment:
you used this here https://github.com/JohnStarich/ee461l-ui/pull/15/files#diff-79368c7cf2861f06556c48578173bed3R8